Lib9c/Miner.cs

```diff
@@ -47,37 +47,47 @@ public Transaction<PolymorphicAction<ActionBase>> StageProofTransaction()
             var txs = new HashSet<Transaction<PolymorphicAction<ActionBase>>>();
             var invalidTxs = txs;
 
-            Transaction<PolymorphicAction<ActionBase>> authProof = null;
             Block<PolymorphicAction<ActionBase>> block = null;
             try
             {
-                if (AuthorizedMiner)
+                if (!AuthorizedMiner)
                 {
-                    authProof = StageProofTransaction();
+                    block = await _chain.MineBlock(
+                        Address,
+                        DateTimeOffset.UtcNow,
+                        cancellationToken: cancellationToken,
+                        maxTransactions: maxTransactions,
+                        append: false);
                 }
-                block = await _chain.MineBlock(
-                    Address,
-                    DateTimeOffset.UtcNow,
-                    cancellationToken: cancellationToken,
-                    maxTransactions: maxTransactions,
-                    append: false);
-
-                if (authProof is Transaction<PolymorphicAction<ActionBase>> proof &&
-                    !block.Transactions.Contains(proof))
+                else
                 {
-                    // For any reason, if the proof tx is not contained mine a new block again
-                    // without any transactions except for the proof tx.
+                    Transaction<PolymorphicAction<ActionBase>> authProof = StageProofTransaction();
                     block = Block<PolymorphicAction<ActionBase>>.Mine(
```

Comment:
It isn't from this PR, but appending with this `block` will fail because it doesn't have a state root hash, (see also: https://github.com/planetarium/libplanet/issues/1345) 😢